src/Microsoft.AspNetCore.Razor.Language/Extensions/SectionTargetExtension.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Language.CodeGeneration;
+
+namespace Microsoft.AspNetCore.Razor.Language.Extensions
+{
+    public sealed class SectionTargetExtension : ISectionTargetExtension
+    {
+        private static readonly string DefaultWriterName = "__razor_section_writer";
```

Comment:
No. This is turd that will go away when we do multitargeting. I don't want it on the public API